mod.yaml

```diff
@@ -19,6 +19,7 @@ Packages:
 	$yupgi_alert: yupgi_alert
 	$ra: ra
 	$cnc: cnc
+	$as: as
```

Comment:
The mod template way of doing things will be to put `OpenRA.Mods.AS.dll` in the `yupgi_alert` directory.

Referencing mods outside the local mods directory (this includes `ra`, `cnc`, etc as well as `as`) is supported via a configuration flag during development, but this must be disabled before installers can be generated because those mods cannot be guaranteed to exist.